python/py-upt-macports/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+set git_hash        20b55dcc6a06f48da8a22c415bc74315dda5cb2e
+github.setup        macport upt-macports ${git_hash}
```

Comment:
you are missing the ```s``` here.... it should be ```github.setup        macports upt-macports ${git_hash}```. Fixing that typo will also simplify the rest of the Portfile as you don't need to set several things anymore  (see below) since the github PortGroup takes care of that...

Reply:
Sorry, for such a silly mistake.